src/expressions/antlr-interface/Expr.g4

```diff
@@ -25,13 +25,14 @@ expr
     | expr op=('/' | '*') expr                 # muldiv
     | expr op=('+' | '-') expr                 # addsub
     | expr COMPARISON expr                     # comparison
-    | 'sum' '(' expr ')'                    # allTimeSum
+    | 'sum' '(' expr ')'                       # allTimeSum
+    | 'sum_connections' '(' expr ')'           # portFieldSum
```

Comment:
if sum_connections can take only port, enforce the rule by doing:
```suggestion
    | 'sum_connections' '(' portField ')'           # sumConnections
```
/!\ i suggest to rename the rule sumConnections